src/index.html

```diff
@@ -550,6 +565,58 @@ <h4>Board and Pieces</h4>
 					<form id="white_squares_form">White squares<br /></form>
 					<br />
 					<form id="black_squares_form">Black squares<br /></form>
+					<br />
+					<form>
```

Comment:
Why are there so many form elements? Seems like every field is wrapped in a separate form.

Reply:
@gruppler yeah it's weird not sure why that's how it was originally so just following suit at least until the client gets rebuilt one of these days

Reply:
I forgot the original thing I wanted to say: It would be good to wrap the labels in `<label>` tags so clicking them toggles the checkbox.

Reply:
good idea, added those.